docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,32 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
 5. any Persistent Disks associated with your cluster
 
 While we hope this list is comprehensive, there could be some lingering resources used by Google Cloud and not deleted automatically that we're not aware of.
+
+Benchmarking the DeepCell Kiosk
+---------------------------------------------------------
+
+The DeepCell Kiosk comes with a utility for benchmarking the scalability and performance of a deep learning workflow. To reproduce the cost and timing benchmarks reported in the DeepCell Kiosk paper, please refer to `the 2020-Bannon_et_al-Kiosk folder of our figure creation repository <https://github.com/vanvalenlab/publication-figures/tree/master/2020-Bannon_et_al-Kiosk>`_. To run your own benchmarking, please read below.
+
+1. If you don't already have a cloud storage bucket for use with the DeepCell Kiosk, you should create one now. It's fine to reuse this bucket across multiple DeepCell Kiosk clusters.
+
+2. There are three variables in the benchmarking pod's YAML file, ``conf/helmfile.d/0410.benchmarking.yaml``, that may need to be customized before benchmarking:
+
+    - ``MODEL`` is the model name and version that will be used in benchmarking. The model you choose should be present in the ``models/`` folder of your benchmarking bucket. See the `Van Valen Lab's benchmarking bucket <https://console.cloud.google.com/storage/browser/kiosk-benchmarking>`_ for an example.
+    - ``FILE`` is the name of the file that will be used for benchmarking. A file by this name should be in your benchmarking bucket in the ``uploads/`` folder.    
+    - ``COUNT`` specifies how many times the ``FILE`` will be submitted to the cluster for processing.
+
+3. Deploy a DeepCell Kiosk as you normally would. While navigating the cluster configuration menu, pay special attention to two configuration settings:
+
+    - The bucket name you provide should be that of the benchmarking bucket from step 1.
+    - The Maximum Number of GPUs has a strong effect on benchmarking time by effectively limiting how large the cluster can scale.
+
+4. Once the cluster has deployed successfully, drop to the ``Shell`` via the DeepCell Kiosk main menu and begin the benchmarking process by executing the following command: ``kubectl scale deployment benchmarking --replicas=1``.
+
+5. Benchmarking jobs can take a day or more, depending on the conditions (# of images and max # of GPUs) chosen. To monitor the status of your benchmarking job, drop to the ``Shell`` within the DeepCell Kiosk main menu and execute the command ``stern benchmarking -s 10m``. This will show you the most recent log output from the `benchmarking` pod. When benchmarking has finished, the final line in the log should be ``Uploaded [FILEPATH] to [BUCKET] in [SECONDS] seconds.``, where ``[FILEPATH]`` is the location in ``[BUCKET]`` where the benchmarking data has been saved.
```

Comment:
are we ok with being informal here? (# instead of number)

Reply:
Good call. I changed the symbols to words, to err on the side of professionalism.